src/numba_stats/poisson.py

```diff
@@ -18,31 +20,40 @@
     Expected value.
 """
 
-
-@_jit(1)
-def _logpmf(k, mu):
+signatures = [
+    nb.float64(nb.int64, nb.float64),
+    nb.float64(nb.uint64, nb.float64),
+    nb.float64(nb.int32, nb.float64),
+    nb.float64(nb.uint32, nb.float64),
+    nb.float64(nb.float64, nb.float64),
+    nb.float32(nb.float32, nb.float32),
+    nb.float32(nb.int32, nb.float32),
+    nb.float32(nb.uint32, nb.float32),
+]
```

Comment:
The number of signatures is intentionally limited so that the compilation does not take too long.